pkg/pgmodel/sql_ingest.go

```diff
@@ -30,6 +31,10 @@ const (
 	getEpochSQL              = "SELECT current_epoch FROM " + catalogSchema + ".ids_epoch LIMIT 1"
 )
 
+var (
+	noOpIngestError = errors.New("trying to ingest new data in read-only mode")
```

Comment:
I think `fmt.Errorf()` might be preferred as that's used mostly across the codebase.